Source/WebCore/platform/graphics/cocoa/WebCoreCALayerExtras.h

```diff
@@ -27,7 +27,7 @@
 
 @interface CALayer (WebCoreCALayerExtras)
 
-+ (CALayer *)_web_renderLayerWithContextID:(uint32_t)contextID;
++ (CALayer *)_web_renderLayerWithContextID:(uint32_t)contextID shouldPreserveFlip:(bool)preservesFlip;
```

Comment:
Argument should be the Objective-C type `BOOL` not `bool`

Reply:
Fixed.